hw_numpy/numpy_challenge.py

```diff
@@ -0,0 +1,43 @@
+if __name__ == "__main__":
+  import numpy as np
+  a = np.ones((3, 5))
+  b = np.arange(0, 60, 5).reshape(3, 4)
+  c = np.random.random((10, 10))
+
+# matrix multiplication
+def matrix_multiplication(first_arr, second_arr):
+   if first_arr.shape[1] == second_arr.shape[0]:
+     return(np.dot(first_arr, second_arr))
```

Comment:
В питоне не надо писать скобочки у `return`

```suggestion
     return(np.dot(first_arr, second_arr))
```